### PR TITLE
Case 22513: Selection JSDoc

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Namepaces:
- Selection

Manuscript case: https://highfidelity.manuscript.com/f/cases/22513/JSDoc-Selection

Note: No executable code changes.

### CODE REVIEW

File: interface/src/scripting/SelectionScriptingInterface.cpp [hunk 1]
```diff
@@ -44,13 +44,14 @@ SelectionScriptingInterface::SelectionScriptingInterface() {
 }
 
 /**jsdoc
+ * The type of an item in a selection list.
```
Comment (jennaingersoll) [inline interface/src/scripting/SelectionScriptingInterface.cpp]:
```suggestion
 * The type of item in a selection list.
```

Comment (jennaingersoll) [reply interface/src/scripting/SelectionScriptingInterface.cpp]: Or -- The type of a specific item in a selection list.

File: interface/src/scripting/SelectionScriptingInterface.h [hunk 2]
```diff
@@ -127,121 +125,119 @@ class SelectionScriptingInterface : public QObject, public Dependency {
...
     /**jsdoc
-    * Remove all items from a selection.
-    * @function Selection.clearSelectedItemsList
-    * @param {string} listName - The name of the selection list.
-    * @returns {boolean} <code>true</code> if the item was successfully cleared, otherwise <code>false</code>.
-    */
+     * Removes all items from a selection list.
+     * @function Selection.clearSelectedItemsList
+     * @param {string} listName - The name of the selection list.
+     * @returns {boolean} <code>true</code> zlways.
```
Comment (jennaingersoll) [inline interface/src/scripting/SelectionScriptingInterface.h]:
```suggestion
     * @returns {boolean} <code>true</code> always.
```

File: interface/src/scripting/SelectionScriptingInterface.cpp (hunk 1 above)
Comment (ctrlaltdavid) [reply interface/src/scripting/SelectionScriptingInterface.cpp]: Done.

File: interface/src/scripting/SelectionScriptingInterface.h (hunk 2 above)
Comment (ctrlaltdavid) [reply interface/src/scripting/SelectionScriptingInterface.h]: Fixed.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19600/HighFidelity-Beta-PR15614-2c591ba.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19600/HighFidelity-Beta-Interface-PR15614-2c591ba.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19600/HighFidelity-Beta-PR15614-2c591ba.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19600/HighFidelity-Quest-Beta-PR15614-2c591ba.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19600/HighFidelity-Beta-PR15614-2c591ba.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19600/HighFidelity-Beta-Interface-PR15614-2c591ba.exe

**Pull request merged**